android/libraries/rib-base/src/main/kotlin/com/uber/rib/core/BasicInteractor.kt

```diff
@@ -22,5 +22,5 @@ package com.uber.rib.core
  * @param <R> the type of [Router].
  */
 abstract class BasicInteractor<P : Any, R : Router<*>> protected constructor(
-  @JvmField protected open var presenter: P
+  protected open var presenter: P
```

Comment:
Is this still binary compatible? Does it still integrate into the amount of repo without (or limited to a handful) issues?

Reply:
It is not binary compatible. However the current bug that was fixed in Kotlin created invalid JVM code that resulted in a runtime crash. This moves the crash to compile time, so as long as build succeeds, it should be fine.

https://youtrack.jetbrains.com/issue/KT-32753

Build and test of major apps against the upgraded version in Android repo was successful.